sdcm/utils/health_checker.py

```diff
@@ -43,8 +43,12 @@ def check_nulls_in_peers(gossip_info, peers_details, current_node):
 
         is_target = current_node.print_node_running_nemesis(ip)
         message = f"Current node {current_node.ip_address}. Found nulls in system.peers for " \
-                  f"node {ip}{is_target} with status {gossip_info[ip]['status']} : " \
-                  f"{peers_details[ip]}"
+            f"node {ip}{is_target} with status {gossip_info[ip]['status']} : " \
+            f"{peers_details[ip]}"
+
+        # By Asias request: https://github.com/scylladb/scylla/issues/6397#issuecomment-666893877
+        LOGGER.debug(f'Print all columns from system.peers for peer {ip}')
+        current_node.run_cqlsh("select * from system.peers where peer = '{self.ip_address}'", split=True, verbose=True)
```

Comment:
Which part of code responsible for the `{self.ip_address}` substitution? Should it be `current_node.ip_address`? Is it a f-string?

Reply:
copy-paste mistake. thanks